master/internal/kubernetes/pods.go

```diff
@@ -324,6 +346,14 @@ func (p *pods) handleGetAgentsRequest(ctx *actor.Context) {
 func (p *pods) summarize(ctx *actor.Context) map[string]agent.AgentSummary {
 	podHandlers := make([]*actor.Ref, 0, len(p.podNameToPodHandler))
 	for _, podHandler := range p.podNameToPodHandler {
+		// Skip pod actors that have not yet received a pod update
```

Comment:
Is this still true?

Reply:
good call, we don't need this anymore now that the pod actor doesn't block on creating the pod. I am going to just drop the commit that made this change.